deployability/modules/allocation/aws/provider.py

```diff
@@ -39,18 +41,45 @@ def _create_instance(cls, base_dir: Path, params: CreationPayload, config: AWSCo
         temp_id = cls._generate_instance_id(cls.provider_name)
         temp_dir = base_dir / temp_id
         credentials = AWSCredentials()
+        teams = ['qa', 'core', 'framework', 'cicd', 'frontend', 'operations', 'cloud', 'threat-intel', 'marketing', 'documentation']
```

Comment:
Use `devops` instead of `cicd` for our team, please.

Reply:
Done: f8244cea4